src/hooks/use-import-export.tsx

```diff
@@ -92,6 +92,18 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 							'The ZIP archive is invalid. Try to unpack and pack it again. If this problem persists, please contact support.'
 						),
 					} );
+				} else if (
+					( error as Error ).message.includes( 'WP-CLI command was canceled (timed out)' )
+				) {
+					await getIpcApi().showErrorMessageBox( {
+						title: __( 'Failed importing site' ),
+						message: sprintf(
+							__(
+								'The import process timed out after %d hours, which can occur when processing very large imports. If the issue persists, please contact support.'
+							),
+							6
```

Comment:
Importing the constant from `wp-cli-process.ts` was breaking the electron app in a weird way, complaining about webpack 5 and I couldn't figure out why. So, for now, I am hardcoding it and we should consider having a constants file like you mentioned.

Reply:
We could use our existing `src/constants.ts` file.